docs/administration/Build_in_docker.md

```diff
@@ -9,22 +9,27 @@ docker pull starrocks/dev-env:{version}
 
 ## Version
 
-| starrocks branch | image tag              |
-| ---------------- | ---------------------- |
-| main             | starrocks/dev-env:main |
-| ...              | ...                    |
+| branch-name      | image-name                    |
+| ---------------- | ----------------------------- |
+| main             | starrocks/dev-env:main        |
+| branch-2.5       | starrocks/dev-env:branch-2.5  |
+| branch-2.4       | starrocks/dev-env:branch-2.4  |
+| branch-2.3       | starrocks/dev-env:branch-2.3  |
 
 ## How to use
 
 - Run the container as usual
 
   ```shell
-  docker run -it --name {container-name} -d starrocks/dev-env:{version}
-  docker exec -it {container-name} /bin/bash
-  # Run git clone starrocks in any path which in the container
+  # mount the docker and login
+  docker run -it --name {branch-name} -d starrocks/dev-env:{branch-name}
```

Comment:
```suggestion
  docker run -it --name sr-{branch-name} -d starrocks/dev-env:{branch-name}
```
will it be better

Reply:
branch-name is ok. Adding a prefix may be misunderstood by a new user.